setup.py

```diff
@@ -10,27 +10,33 @@
 def get_decoder_pkg_url(version, artifacts_root=None):
     is_arm = 'arm' in platform.machine()
     is_mac = 'darwin' in sys.platform
+    is_win = 'win32' in sys.platform
```

Comment:
`sys.platform` return `win32` even on 64-bits platforms? surprising :)

Reply:
Yep, was surprising to me too.